packages/ens-referrals/src/referrer.ts

```diff
@@ -0,0 +1,56 @@
+import { type Address, bytesToHex, type Hex, hexToBytes, pad, zeroAddress } from "viem";
+
+/**
+ * Encoded Referrer
+ *
+ * Guaranteed to be a string representation of 32-bytes.
```

Comment:
specify whether the 20byte address is padded left or right in this semantic type